cmd/nebula-cert/sign.go

```diff
@@ -230,6 +230,13 @@ func x25519Keypair() ([]byte, []byte) {
 	return pubkey[:], privkey[:]
 }
 
+func x25519PubKey(privkey []byte) []byte {
+	var pubkey, priv32 [32]byte
+	copy(priv32[:], privkey)
+	curve25519.ScalarBaseMult(&pubkey, &priv32)
```

Comment:
we should update this to use `curve25519.X25519` like we did elsewhere https://github.com/slackhq/nebula/pull/533